tests/testsuite/namespace.go

```diff
@@ -200,12 +201,23 @@ func removeNamespaces() {
 
 	// Wait until the namespaces are terminated
 	fmt.Println("")
-	for _, namespace := range TestNamespaces {
-		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
+	for _, namespaceName := range TestNamespaces {
+		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespaceName)
 		EventuallyWithOffset(1, func() error {
-			return virtCli.CoreV1().Namespaces().Delete(context.Background(), namespace, metav1.DeleteOptions{})
-		}, 240*time.Second, 1*time.Second).Should(SatisfyAll(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())), fmt.Sprintf("should successfully delete namespace '%s'", namespace))
+			return virtCli.CoreV1().Namespaces().Delete(context.Background(), namespaceName, metav1.DeleteOptions{})
+		}, 240*time.Second, 1*time.Second).Should(SatisfyAll(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())), fmt.Sprintf("should successfully delete namespace '%s': \n%s", namespaceName, dumpNamespaceObject(namespaceName)))
+	}
+}
+
+func dumpNamespaceObject(namespaceName string) string {
+	virtCli, err := kubecli.GetKubevirtClient()
+	util.PanicOnError(err)
+	ns, _ := virtCli.CoreV1().Namespaces().Get(context.Background(), namespaceName, metav1.GetOptions{})
```

Comment:
basically the Get might failed here, please explicit add verification (maybe the err check on 217 is due to this fact, but imo it is better to be explicit here as well)

Reply:
Done

Reply:
Note: we just need to know that if it happens, it means that waiting more would be ok